### PR TITLE
[ELY-2591] Move min method into the PeriodicRotatingFileAuditEndpoint…

### PR DESCRIPTION
**Pull request by gerjantd**
…#Builder class

https://issues.redhat.com/browse/ELY-2591

### CODE REVIEW

**Review by PrarthonaPaul — APPROVED** [review]
LGTM!

**Review by skyllarr — APPROVED** [review]

File: audit/src/main/java/org/wildfly/security/audit/PeriodicRotatingFileAuditEndpoint.java [hunk 1]
```diff
@@ -174,6 +174,10 @@ public static class Builder extends FileAuditEndpoint.Builder {
             super();
         }
 
+        private static <T extends Comparable<? super T>> T min(T a, T b) {
```
Comment (skyllarr) [inline audit/src/main/java/org/wildfly/security/audit/PeriodicRotatingFileAuditEndpoint.java]: @gerjantd Just a total minor, can  you please place this method below the public methods in the Builder class for convention reasons? Thank you!

Comment (gerjantd) [reply audit/src/main/java/org/wildfly/security/audit/PeriodicRotatingFileAuditEndpoint.java]: @Skyllarr  done. I squashed the commits. Do you need a new PR or can you take it from here?

**Review by darranl — APPROVED** [review]

**Comment by darranl** [comment]
Thank you @gerjantd

**Pull request merged**